charts/gardener/controlplane/charts/runtime/templates/apiserver/hvpa.yaml

```diff
@@ -0,0 +1,73 @@
+{{- if .Values.global.apiserver.hvpa.enabled }}
+{{- /* .Values.global.apiserver.replicaCount is of type 'float64', so let's cast it to string to have proper types for comparison */}}
+{{- if ne (.Values.global.apiserver.replicaCount | toString) "0" }}
+apiVersion: autoscaling.k8s.io/v1alpha1
+kind: Hvpa
+metadata:
+  name: gardner-apiserver-hvpa
+  namespace: garden
+spec:
+  replicas: 1
+{{- if .Values.global.apiserver.hvpa.scaleUpStabilization }}
+  scaleUpStabilization:
+{{ toYaml .Values.global.apiserver.hvpa.scaleUpStabilization | indent 4 }}
+{{- end }}
+{{- if .Values.global.apiserver.hvpa.scaleDownStabilization }}
+  scaleDownStabilization:
+{{ toYaml .Values.global.apiserver.hvpa.scaleDownStabilization | indent 4 }}
+{{- end }}
+  hpa:
+    selector:
+      matchLabels:
+        role: gardener-apiserver-hpa
+    template:
+      metadata:
+        labels:
+          role: gardener-apiserver-hpa
+      spec:
+        maxReplicas: {{ .Values.global.apiserver.hvpa.maxReplicas }}
+        minReplicas: {{ .Values.global.apiserver.hvpa.minReplicas }}
+        metrics:
+        - resource:
+            name: memory
+            targetAverageUtilization: {{ .Values.global.apiserver.hvpa.targetAverageUtilization }}
```

Comment:
I am not sure, but would it make sense to be able to configure the `targetAverageUtilization` separately for cpu and memory?